packages/user-operation-controller/src/helpers/SnapSmartContractAccount.ts

```diff
@@ -92,6 +98,9 @@ export class SnapSmartContractAccount implements SmartContractAccount {
       'KeyringController:signUserOperation',
       sender,
       userOperation,
+      {
+        chainId: request.chainId,
```

Comment:
Same for these three, can we destructure them when we first access `request` at the start of the method?